docs/source/release_notes.rst

```diff
@@ -5,15 +5,19 @@ Release Notes
         * Increased the amount of logical type information passed to Woodwork when calling ``ww.init()`` in transformers :pr:`3604`
         * Added ability to log how long each batch and pipeline take in ``automl.search()`` :pr:`3577`
         * Added the option to set the ``sp`` parameter for ARIMA models :pr:`3597`
+        * Added Dockerfile.arm and .dockerignore for python version and M1 testing :pr:`3609`
+        * Added ``test_gen_utils::in_container_arm64()`` fixture :pr:`3609`
```

Comment:
This may fit better under `Testing Changes` unless we plan on adding more information to users to utilize the container.

Reply:
Done